sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/AsynReadWithMultipleClients.java

```diff
@@ -181,7 +181,7 @@ protected void onSuccess() {
     protected void onError(Throwable throwable) {
     }
 
-    private void createClients() {
+    private void createClients(Configuration cfg) {
```

Comment:
We already have the `configuration` object in this class, why do we need to create a separate one ?

Reply:
It was the same object , however redundant reference , removed it now. Thanks